### PR TITLE
Decode bytes input to utf-8 string before passing to vllm engine

### PR DESCRIPTION
**Pull request by rmccorm4**
- Removes nested bytestring of prompt embedded in responses
- Improves outputs - cleaner and more coherent. It was probably a bug to send bytestring to vllm engine.

### CODE REVIEW

File: Quick_Deploy/vLLM/model_repository/vllm/1/model.py [hunk 1]
```diff
@@ -167,6 +167,8 @@ async def generate(self, request):
         try:
             request_id = random_uuid()
             prompt = pb_utils.get_input_tensor_by_name(request, "PROMPT").as_numpy()[0]
+            if isinstance(prompt, bytes):
+                prompt = prompt.decode("utf-8")
```
Comment (tanmayv25) [inline Quick_Deploy/vLLM/model_repository/vllm/1/model.py:171]: Shouldn't need to cast the prompt to str in L189 case?

Comment (rmccorm4) [reply Quick_Deploy/vLLM/model_repository/vllm/1/model.py:171]: done

**Review by dyastremsky — APPROVED** [review]

**Review by nnshah1 — APPROVED** [review]

**Comment by dyastremsky** [comment]
@rmccorm4, out of curiosity, where did you find the "generate" endpoint? I'd love to see the rest of the curl commands. That one seems mighty useful.

**Comment by rmccorm4** [comment]
Generate endpoint is at top of tree, merged here: https://github.com/triton-inference-server/server/pull/6366. Docs coming soon.

**Pull request merged**